src/connection-manager/connections.ts

```diff
@@ -97,7 +99,28 @@ async function setHandlers(
         )
       }
     })
-    ws.on('close', () => {
+    ws.on('close', async (code, reason) => {
+      if (connectionsInitialized) {
+        log.error(
+          `Websocket closed for ${ws.url} on ${
+            networks ?? 'unknown network'
```

Comment:
`networks ?? 'unknown network'` should be a variable for ease of use/readability